PostFreeSurfer/scripts/GenerateStructuralScenes.sh

```diff
@@ -32,8 +32,8 @@ PARAMETERs are [ ] = optional; < > = user supplied value
 
 #arguments to opts_Add*: switch, variable to set, name for inside of <> in help text, description, [default value if AddOptional], [compatibility flag, ...]
 #help info for option gets printed like "--foo=<$3> - $4"
-opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects"
-opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all sessions"
+opts_AddMandatory '--session' 'Session' 'session ID' ""
```

Comment:
```suggestion
opts_AddMandatory '--session' 'Session' 'session ID' "" '--subject'
```

Reply:
Actually, since it is in a `/scripts/` folder, legacy flag support probably isn't needed.